src/main/java/com/salesforce/storm/spout/sideline/trigger/SidelineTrigger.java

```diff
@@ -38,8 +39,18 @@
      * Set the sideline spout trigger's proxy on the trigger.
      * @param spout Sideline spout trigger's proxy
```

Comment:
add a deprecated tag to the javadoc and explain what its replaced with, or why you shouldn't use it,

Reply:
I went ahead and dropped this entirely.